### PR TITLE
feat(combined-report-ui): add "How to fix" card row

### PR DESCRIPTION
**Pull request by pownkel**
#### Description of changes

Add missing "how to fix" card row to combined report. This involves changing the input interface to accept the resolution data that we get from axe-core.

howToFixCard

#### Pull request checklist

- [x] Addresses an existing issue: #1772544
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by JGibson2019** [comment]
Figma clarifications:

- Should there be an unordered list in the URL row?

- Noticed that there are grey lines underneath each row; should those be removed to match the Figma? 
image

**Comment by dbjorge** [comment]
> Should there be an unordered list in the URL row?
This is intentional, per earlier discussion with @ferBonnin 

> Noticed that there are grey lines underneath each row; should those be removed to match the Figma?
The gray separators are consistent with all other card views; I think we should leave them for consistency, but good to double check with @ferBonnin

**Review by JGibson2019 — COMMENTED** [review]
LGTM, but wanted to check on the styling things before fully approving

**Comment by JGibson2019** [comment]
> > Should there be an unordered list in the URL row?
> 
> This is intentional, per earlier discussion with @ferBonnin
> 
> > Noticed that there are grey lines underneath each row; should those be removed to match the Figma?
> 
> The gray separators are consistent with all other card views; I think we should leave them for consistency, but good to double check with @ferBonnin

Re 1 - Thank you! 
Re 2 - I'm also a fan of keeping them for consistency's sake

**Comment by ferBonnin** [comment]
> > Noticed that there are grey lines underneath each row; should those be removed to match the Figma?
> 
> The gray separators are consistent with all other card views; I think we should leave them for consistency, but good to double check with @ferBonnin


yes, we should leave the grey separators for consistency.

**Review by JGibson2019 — APPROVED** [review]

File: src/reports/package/accessibilityInsightsReport.d.ts [hunk 1]
```diff
@@ -61,11 +61,18 @@ declare namespace AccessibilityInsightsReport {
         ruleUrl: string
     }
 
+    export type HowToFixData = {
+        any: FormattedCheckResult[],
```
Comment (dbjorge) [inline src/reports/package/accessibilityInsightsReport.d.ts:65]: `FormattedCheckResult` doesn't seem to be defined or explicitly imported from this file (I think it's coming implicitly from `scanner.d.ts`, so I'm not sure it's going to be usable for a user of the package. It's a pretty short declaration so I'd lean towards just repeating it here.

**Review by dbjorge — CHANGES_REQUESTED** [review]
One suggestion, looks good overall!

**Review by dbjorge — APPROVED** [review]

**Pull request merged**